server/routes/nl/api.py

```diff
@@ -80,3 +83,41 @@ def feedback():
   except Exception as e:
     logging.error(e)
     return 'Failed to record feedback data', 500
+
+
+@bp.route('/encode-vector')
+@file_cache.file_cache.cached(timeout=cache.TIMEOUT, query_string=True)
+def encode_vector():
+  if not current_app.config['VERTEX_AI_MODELS']:
+    flask.abort(404)
+  sentence = request.args.get('sentence')
+  model_name = request.args.get('modelName')
```

Comment:
Document where/how to get the valid `modelName` values?

Reply:
Done